sanic_jwt/blueprint.py

```diff
@@ -72,14 +79,21 @@ def get_token_reponse(request, access_token, output, refresh_token=None):
 
     try:
         payload = request.app.auth.extract_payload(request)
-        user = await request.app.auth.retrieve_user(request, payload)
+        user = request.app.auth.retrieve_user(request, payload)
+        if inspect.isawaitable(user):
+            user = await user
```

Comment:
Same. We should probably throw an exception if `retrieve_user` is not implemented.

I guess this brings up another question of whether these checks belong here, or if they should be done at the initialization stage.

Reply:
This will be handled by `Initialize`, right?

Reply:
Yes.